src/Avalonia.Input/InputElement.cs

```diff
@@ -347,7 +347,7 @@ public Cursor Cursor
         public bool IsFocused
         {
             get { return _isFocused; }
-            private set { SetAndRaise(IsFocusedProperty, ref _isFocused, value); }
+            set { SetAndRaise(IsFocusedProperty, ref _isFocused, value); }
```

Comment:
@Sorien  have you tried making this setter internal then adding an `InternalIsVisibleTo` to whatever internal assembly needs setting this?

Reply:
nop, all 3rd party controls that contain any kind of edit (masked edits, tag editors and so on) will need s/t like that so the q is should these controls have their own property like "IsTextBoxInsideFocused" or should we use private IsFocused that is there just for `:focus` pseudoclass